telemetry/csharp/AwsToolkit.Telemetry.Events.Generator.Tests/test-data/expectedCode-supplemental.txt

```diff
@@ -56,9 +56,17 @@ namespace Test
                 datum.AddMetadata("awsAccount", payload.AwsAccount);
                 datum.AddMetadata("awsRegion", payload.AwsRegion);
 
-                if ((transformDatum != null))
+                try
```

Comment:
Another option would be to move this code to a new helper class in telemetry/csharp/AwsToolkit.Telemetry.Events/Core and make a static method that accepts a MetricDatum list and a transform, and returns a MetricDatum list, and then you would only need to generate the call to this method.

I'm okay with this current approach too.

Reply:
Updated - I like this idea.